src/styles/blocks/card.scss

```diff
@@ -0,0 +1,81 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: $width-card;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition: all 300ms;
+
+  &__image {
+    align-self: center;
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    @include styleFont(12px, 500, 18px);
+  }
+
+  &__code {
+    margin-bottom: 16px;
+    @include styleFont(10px, 400, 14px);
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__reviews-count {
+    @include styleFont(10px, 400, 14px);
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    &-text {
+      @include styleFont(12px, 400, 18px);
+    }
+
+    &-sum {
+      @include styleFont(16px, 700, 18px);
+    }
+  }
+
+  &__link {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
+    width: 166px;
+    @include styleFont(14px, 700, 16px);
+    text-transform: uppercase;
+    text-decoration: none;
+    background-color: $hover-color;
+    color: #fff;
+    border: 1px solid $hover-color;
+    border-radius: 5px;
+
+    &:hover {
+      background-color: #fff;
+      color: #060b35;
```

Comment:
Would be great to add transition for these properties too. Also, the color is wrong
![image](https://github.com/mate-academy/layout_catalog/assets/96409563/9cfc7317-779e-422b-a03d-f909364b94a3)